### ISSUE TITLE
Sentry is just blocking RAM  and Resources without doing nothing.

### ISSUE THREAD

**Issue by ecxod** [getsentry/self-hosted#2851]
### Self-Hosted Version

24.2.0

### CPU Architecture

x86_64

### Docker Version

20.10.5+dfsg1

### Docker Compose Version

1.25.0

### Steps to Reproduce

`./install.sh`

After starting all containers it is using 8.2 GB RAM for nothing. No Bug. No entry. It is simply occupied a lot of RAM. At the end of the day we pay for resources. 
Slowly understand your logic. Self-hosting should cost more than Saas.

### Expected Result

use less RAM

### Actual Result

use less RAM

**Comment by aldy505** [comment]
Yeah this is a known issue, and it's a bummer for sure. Sentry's on the way for migrating some of their stuff to something that's memory efficient. One way of it is to migrate the current Snuba consumers that are written in Python to be rewritten in Rust. It should be available on the March release (15th of March), see the PR here: https://github.com/getsentry/self-hosted/pull/2831.

Another thing that's supposed to be taking lots of RAM usage is Kafka and Zookeeper, you can swap it with Redpanda, it works, but the risk is on your own as it won't be officially supported by Sentry employees if you encounter any issues with it.

If you still don't like that it uses lots of resources for that much feature doing nothing, perhaps you can consider bumping this issue: https://github.com/getsentry/self-hosted/issues/2773, it'll only enable the errors feature, so no performance, metrics, replays and profiling. That would lessen the containers in use, and therefore consume less resource.

I'm closing this issue as there are a few issues that similar to this one. See https://github.com/getsentry/self-hosted/issues/1719, https://github.com/getsentry/self-hosted/issues/1763
